src/main/java/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,45 @@
+package step2;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    private static String DEFAULT_REGEX_DELIMITER = ",|:";
+    private static String CUSTOM_REGEX_DELIMITER = "//(.)\n(.*)";
```

Comment:
상수로 선언하는 걸 의도하신 것으로 보이는데, `final` 키워드가 빠진 것 같습니다.
```suggestion
    private static final String DEFAULT_REGEX_DELIMITER = ",|:";
    private static final String CUSTOM_REGEX_DELIMITER = "//(.)\n(.*)";
```